### DISCUSSION TITLE
Is there any way to change background color for different notebooks?

### DISCUSSION THREAD

**Discussion by quesada** [zim-desktop-wiki/zim-desktop-wiki#2073] [Ideas]
I have a notebook per section of the company, and would like to avoid adding content to the wrong notebook.
One way to do this is to have different colors. Is this possible?

**Comment by Callisos** [comment]
I'd like that feature too, though for a different reason: When on Linux/KDE switching the application style for GTK apps to the "Breeze-Dark" theme, the background color of wiki pages in Zim turns to black and the text turns to white.

This causes a problem with readability of highlighted and verbatim text:
![Zim⁄Breeze-Dark](https://user-images.githubusercontent.com/12926163/109047839-a1872680-76d6-11eb-885f-1edbc9eeed18.png)

When having a possibility of changing wiki pages background color, I could enjoy unified dark-themed apps along with Zim while having a white-colored Zim pages with well readable text on them.

**Comment by m0tt** [comment]
> I'd like that feature too, though for a different reason: When on Linux/KDE switching the application style for GTK apps to the "Breeze-Dark" theme, the background color of wiki pages in Zim turns to black and the text turns to white.
> 
> This causes a problem with readability of highlighted and verbatim text:
> ![Zim⁄Breeze-Dark](https://user-images.githubusercontent.com/12926163/109047839-a1872680-76d6-11eb-885f-1edbc9eeed18.png)
> 
> When having a possibility of changing wiki pages background color, I could enjoy unified dark-themed apps along with Zim while having a white-colored Zim pages with well readable text on them.

Indeed this issue is very annoying. Highlighted text in dark mode is not readabale at all.

**Comment by sanjarcode** [comment]
This is indeed a badly needed feature. A notebook-level configuration file for styles would be good.
I tried to add styles to `notebook.zim` as contained in `~/.config/zim/style.conf`, but that doesn't work.

What do you think @jaap-karssenberg?

**Comment by mhar-vell** [comment]
> This is indeed a badly needed feature. A notebook-level configuration file for styles would be good.
> I tried to add styles to `notebook.zim` as contained in `~/.config/zim/style.conf`, but that doesn't work.
> 
> What do you think @jaap-karssenberg?

Did you get any solution?

**Comment by jaap-karssenberg** [comment]
Background color can be controlled via the gtk theme. Please see
documentation in the manual about css to modify it specific for Zim. That
should be sufficient for fixing issues with dark theme.

Changing it per notebook would require a plugin that sets the css per
window.



Op vr 4 jun. 2021 03:11 schreef Marco Reis ***@***.***>:

**Reply by introt** [reply]
See also: #720
